packages/react-components/react-card/src/components/Card/useCardStyles.ts

```diff
@@ -36,82 +49,132 @@ const useStyles = makeStyles({
   filledInteractive: {
     cursor: 'pointer',
     backgroundColor: tokens.colorNeutralBackground1,
-    ...shorthands.borderColor(tokens.colorTransparentStroke),
     boxShadow: tokens.shadow4,
 
+    '::after': {
+      ...shorthands.borderColor(tokens.colorTransparentStroke),
+    },
+
     ':hover': {
       backgroundColor: tokens.colorNeutralBackground1Hover,
-      ...shorthands.borderColor(tokens.colorTransparentStrokeInteractive),
       boxShadow: tokens.shadow8,
+
+      '::after': {
+        ...shorthands.borderColor(tokens.colorTransparentStrokeInteractive),
```

Comment:
**thought:** wondering if this "reset" back to default state is necessary for pseudo selectors state snapshot. 

maybe we can use something like  `...shorthands.borderColor('inherit'),` ?

Reply:
This isn't a reset to the default state as the default is the token `colorTransparentStroke`.

Reply:
got it, the whole style declaration is bit confusing that what got me wondering I guess. ty